src/browser/BrowserAction.cpp

```diff
@@ -541,7 +541,7 @@ QJsonObject BrowserAction::handlePasskeysGet(const QJsonObject& json, const QStr
     }
 
     const auto origin = browserRequest.getString("origin");
-    if (!origin.startsWith("https://")) {
+    if (!browserSettings()->allowLocalhostWithPasskeys() && !origin.startsWith("https://")) {
```

Comment:
This doesn't just allow localhost, it allows anything. Shouldn't there also be a check to see if the host is localhost?

Reply:
Yes :)